e2e/cypress/index.d.ts

```diff
@@ -8,10 +8,7 @@ declare namespace Cypress {
     restoreStorage(): void
     applyProjectsFilter(projectsToFilterOn: string[]): void
     generateAdminToken(idToken: string): void
-    cleanupPoliciesByIDPrefix(idToken: string, idPrefix: string): void
-    cleanupProjectsByIDPrefix(idToken: string, idPrefix: string): void
-    cleanupRolesByIDPrefix(idToken: string, idPrefix: string): void
-    cleanupTokensByIDPrefix(idToken: string, idPrefix: string): void
+    cleanupV2IAMObjectsByIDPrefixes(idToken: string, idPrefix: string, objectPlurals: string[]): void
```

Comment:
Didn't wanna add n+1 cleanup helper so I squashed them down for all IAM V2 objects 😅

Reply:
nice refactor! someday it'd be cool if we got all of the test objects created with the same prefix. then we could have something even more generic like `deleteCypressTestObjects` (and pass whatever paths)